docs/release_guide.md

```diff
@@ -97,6 +98,20 @@ upload the packaged binaries and checksums as release assets, build and push Doc
 In the description section of the new release, add information about feature additions, bug fixes,
 and any other administrative tasks completed on the repository.
 
+## Update docs site
+
+In the demo site's main branch, edit the file [https://github.com/openservicemesh/osm-docs/blame/main/content/docs/install/manual_demo.md](https://github.com/openservicemesh/osm-docs/blame/main/content/docs/install/manual_demo.md) to update any version references in the manual demo on the docs site.
```

Comment:
If the docs site is also keeping release branches, is the process for creating and updating them with these changes documented anywhere?